automata/symbol/graph/symbol_graph.py

```diff
@@ -40,6 +46,9 @@ def __init__(
         self._graph = builder.build_graph()
         self.navigator = SymbolGraphNavigator(self._graph)
 
+        with open(f"{data_root_path}/symbolgraph.pkl", "wb") as f:
```

Comment:
let's call it symbol_graph.pkl btw, this is consistent with how we go from CamelCase to snake_case.

Reply:
My Java tendencies are showing 😵‍💫 Updated!